python/activator/visit.py

```diff
@@ -48,7 +48,16 @@ class Dome(enum.IntEnum):
     survey: str                 # survey name
```

Comment:
Can you also update the documentation of `filters` to say that these filter names are in the Middleware format, including `~` for composite filters? Following up on https://lsstc.slack.com/archives/C0345PUU4N9/p1679429952829539.